src/Equinor.Procosys.Library.WebApi/Controllers/Register/RegisterController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Equinor.Procosys.Library.Domain;
+using Equinor.Procosys.Library.Query.GetAllRegisters;
+using Equinor.Procosys.Library.WebApi.Misc;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using ServiceResult.ApiExtensions;
+
+namespace Equinor.Procosys.Library.WebApi.Controllers.Register
```

Comment:
'Registers' (if you change controller name)

Reply:
if you refer to namespace, the namespace is not in plural for Area / Disciplione. Wont fix